README.md

```diff
@@ -8,6 +8,14 @@ Born out of my own need to keep track of appartment listings when looking to ren
 
 Get the extension [from the Chrome Web Store](https://chrome.google.com/webstore/detail/mark-as-read/hiflhkmicfagennabmnfcnnlpkmidfjj).
 
+## Features
```

Comment:
Let's move this below the How to Use section

Reply:
Also let's change to:

Features
    Mark pages as read
    Highlight visited text links
    Backup and Restore